ui/ducks/send/send.js

```diff
@@ -660,12 +656,13 @@ export const initializeSendState = createAsyncThunk(
     const isNonStandardEthChain = getIsNonStandardEthChain(state);
     const selectedNetworkClientId = getSelectedNetworkClientId(state);
     const chainId = getCurrentChainId(state);
-    let eip1559support = checkNetworkAndAccountSupports1559(state);
+    let eip1559support =
+      state.metamask.networksMetadata?.[selectedNetworkClientId]?.EIPS?.[1559];
```

Comment:
The existing selector was returning false when it should return undefined for the follow up logic to work.